reana_client/cli/utils.py

```diff
@@ -22,3 +24,42 @@ def add_access_token_options(func):
     def wrapper(*args, **kwargs):
         return func(*args, **kwargs)
     return wrapper
+
+
+def parse_parameters(filter):
```

Comment:
Can you change this variable name here and elsewhere to `filter_` or a different name to prevent shadowing the built-in function?